packages/animations/browser/src/errors.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+const LINE_START = '\n - ';
+
+export function invalidTimingValue(exp: string|number): Error {
+  return new Error(`The provided timing value "${exp}" is invalid.`);
```

Comment:
I think it'd be great to use `RuntimeError` class from the `@angular/core`, so it's more consistent with other packages.

We'd also need to use `ngDevMode` to make error messages (strings) tree-shakable.

```suggestion
  return new RuntimeError(
    RuntimeErrorCode.INVALID_TIMIMG_VALUE,
    NG_DEV_MODE && `The provided timing value "${exp}" is invalid.`);
```
where `NG_DEV_MODE` can be declared at the top of the file as:

```
const NG_DEV_MODE = typeof ngDevMode === 'undefined' || !!ngDevMode;
```

Without the `NG_DEV_MODE`, the build optimizer wouldn't be able to tree-shake the string away.

Reply:
I initially was going to do that, but then saw the reactive forms errors and based it off of that instead. I'll switch over. Thanks for the heads up.